doc/ADDB-User-Guide.md

```diff
@@ -0,0 +1,79 @@
+# ADDB User Guide
+[ADDB](https://github.com/Seagate/cortx-motr/blob/main/doc/ADDB.md) can track how long each step takes during a Motr operation, therefore can furnish valuable information on what procedures serve as the bottleneck of the system. This guide provides an example on how to use ADDB to generate a request timeline graph to analyze an I/O process.
```

Comment:
If I understand it right, Motr has built-in ADDB data points to record timestamps at critical points of an operation execution, ADDB doesn't track the duration of each step directly, the duration of each step is actually calculated by analysing tool such as the one mentioned in this document.